README.md

```diff
@@ -9,6 +9,30 @@ An abstraction layer for low-level platform-specific graphics APIs.
 * Reference documentation: http://wiki.gpfault.net/docs/nicegraf
 * Sample code: https://github.com/nicebyte/nicegraf/tree/master/samples
 
+# prerequisites
+
+## Windows
+
+- CMake 3.14.0 or above
```

Comment:
Cmake is required on all platforms. Also, let's remove the specific version, it's already mentioned in the cmake file, let's not repeat it.